assets/js/modules/analytics/common/account-select.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Analytics Account Select component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { Select, Option } from '../../../material-components';
+import { STORE_NAME, ACCOUNT_CREATE } from '../datastore/constants';
```

Comment:
Shouldn't we take such imports from `../datastore` instead? The `constants` file is to avoid circular dependencies, but that isn't the case here, and the existence of that file is maybe more of a store module-internal thing I'd think, where here it is accessed from the outside.

Reply:
I don't recall exactly what it was but this is where I started looking into circular dependencies in the first place.

We don't currently export `ACCOUNT_CREATE` from the datastore index though so that would have to be exposed there as well otherwise we would need to import from this module anyways.